Testing/TemplateTestCase.m

```diff
@@ -0,0 +1,61 @@
+%% Template Test Case Setup
+%
+% This template will establish the framework for creating test cases
+
+% Create a short name for the test case
+TestCase.Name        = 'Template';
+
+% Write a brief description of what the test case is testing
+TestCase.Description = 'This test case is a template to edit and create new test cases';
+
+% Establish Initial Conditions of the test
+P0 = [0, 10]; % Initial position (m)
+V0 = [-2, 0]; % Initial velocity (m/s)
+
+StateVector_Initial = [P0 V0];
+TestCase.InitialConditions = StateVector_Initial;
+
+% The pitch table will be linearly interpolated using the simulation time as the independent input
+% The first column is time. (Should be monotonically increasing with no repeats)
+% The second column is pitch angle in degrees
+TestCase.PitchTable =  [0 0;
+                        5 0;
+                        5.001 5;
+                        10 5;
+                        10.001 0;
+                        15 0];
+
+% The throttle table will be linearly interpolated using the simulation time as the independent input
+% The first column is time. (Should be monotonically increasing with no repeats)
+% The second column is throttle percentage (1 is 100%)
+TestCase.ThrottleTable = [0 0;
+                          2 1;
+                          15 1];
+% Calculate the stop time of the test. Simply the largest last input time.
+TestCase.StopTime = max([TestCase.ThrottleTable(end,1),TestCase.PitchTable(end,1)]);
+
+%% Run the Test Case through the Aircraft Simulation

```

Comment:
Same thing about new lines.

Reply:
Converted all files to LF, as well as set my default to LF from now on to match the other files.